src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -153,77 +169,156 @@ export default class SimidMapCreative extends BaseSimidCreative {
   playContent_() {
     this.simidProtocol.sendMessage(CreativeMessage.REQUEST_SKIP);
   }
-  
+
   /**
- * Loads a map object that currently defaults to a hardcoded location.
- * @param {!google.maps.LatLng=} coordinates The LatLng object of user's current location.
- * @private 
- */
-displayMap_(coordinates = new google.maps.LatLng(DEFAULT_MAP_LAT, DEFAULT_MAP_LNG)) {
-  this.map_ = new google.maps.Map(document.getElementById('map'), {
-    zoom: DEFAULT_ZOOM,
-    center: coordinates
-  });
-  new google.maps.Marker({
-    position: coordinates,
-    map: this.map_,
-    title: 'Current Position'
-  });
-  this.findNearby_(this.searchQuery_, coordinates);
-}
+   * Loads a map object that currently defaults to a hardcoded location.
```

Comment:
Am I missing something, or does this block of code not do anything different than the above deleted one?

Reply:
You aren't missing anything-- it's exactly the same. Github flagged it as different for whatever reason but it's the same functionality